app/javascript/components/applications/application_form/HouseholdMembersSection.js

```diff
@@ -1,20 +1,15 @@
 import React from 'react'
 import HouseholdMemberForm from './HouseholdMemberForm'
-import { isEmpty } from 'lodash'
 import { FieldArray } from 'react-final-form-arrays'
 import validate from '~/utils/form/validations'
 
 const memberValidate = (values) => {
   if (!values || !values.length) return
   const membersErrors = []
   values.forEach(value => {
-    if (!isEmpty(value)) {
-      const memberError = {date_of_birth: {}}
-      validate.isValidDate(value.date_of_birth, memberError.date_of_birth, {errorMessage: 'Please enter a Date of Birth'})
-      memberError.first_name = validate.isPresent('Please enter a First Name')(value.first_name)
```

Comment:
moved these to the individual field-level validation to make it simpler. Would be great if we could have dob validation on the MultiDateInput component too